test/e2e/README.md

```diff
@@ -2,11 +2,21 @@
 
 ## Run E2E tests Locally
 ### Prerequisite
- - Make sure a kubernetes cluster(with version >= 1.13) is set up and kubeconfig is under `$HOME/.kube/config`
- - Copy out `/etc/kubernetes/azure.json` under one agent node to local machine
- > For AKS cluster, need to modify `resourceGroup` to the node resource group name inside `/etc/kubernetes/azure.json`
-
-### Run test
+- Set up a Kubernetes cluster (with version >= 1.13) using [aks-engine](https://github.com/Azure/aks-engine) or [AKS](https://docs.microsoft.com/en-us/azure/aks/)
+- `$KUBECONFIG` is set or your kubeconfig is under `$HOME/.kube/config`
+- Export the following environment varibles:
+```bash
+export tenantId=<your tenant ID>
+export subscriptionId=<the Azure subscription ID that your cluster is under>
+export aadClientId=<the service principal ID that your cluster is using>
+export aadClientSecret=<the service principal password that your cluster is using>
+export resourceGroup=<the resource group that your cluster is under>
+export location=<the location of your resource group>
 ```
-make e2e-test
+
+To run the E2E tests:
+
+```bash
+docker login # Login to docker so the test image can be pushed to your Docker Hub
+REGISTRY=<your Docker Hub ID> make e2e-test
```

Comment:
how can I use `mcr.microsoft.com/k8s/csi/azurefile-csi:version` to run e2e test?

Reply:
I've made some adjustments to Makefile. You can now do that by executing `REGISTRY=mcr.microsoft.com/k8s/csi IMAGE_VERSION=version make e2e-test`.